### DISCUSSION TITLE
Additional preview environments support using the new Package Manifest 15+

### DISCUSSION THREAD

**Discussion by IbrahimMNada** [umbraco/Umbraco-CMS#17816] [Features and ideas]
This [link ](https://docs.umbraco.com/umbraco-cms/reference/content-delivery-api/additional-preview-environments-support) shows that you can add multi environments. 

but its not working on umbraco15v and the `SendingContentNotification ` class is removed

ref this issue : https://github.com/umbraco/Umbraco-CMS/issues/17035

ref this commit : https://github.com/umbraco/Umbraco-CMS/blob/bff321e6f5999e162210a8d36cf077ce54998966/src/Umbraco.Web.UI.Client/src/packages/documents/documents/workspace/actions/save-and-preview.action.ts#L7

lets add a manifiest with type preview env and handle this issue

**Comment by nul800sebastiaan** [comment]
Thanks, we're aware, please subscribe to updates on https://github.com/umbraco/Umbraco-CMS/issues/17035
